shell/browser/electron_browser_context.cc

```diff
@@ -679,4 +718,21 @@ ElectronBrowserContext* ElectronBrowserContext::From(
   return new_context;
 }
 
+ElectronBrowserContext* ElectronBrowserContext::FromPath(
+    const base::FilePath path,
```

Comment:
It does not match the header.

```suggestion
    base::FilePath path,
```